### PR TITLE
osx/asr.md: make placeholders auto convertible to CLIP ones

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- https://github.com/command-line-interface-pages/prototypes/pull/32

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

## Note

`file` suffix is required to make placeholder convertible.

## Result

Conversion result thanks to fixes:

```md
# asr

> Restore (copy) a disk image onto a volume
> The command name stands for Apple Software Restore
> More information: https://www.unix.com/man-page/osx/8/asr/

- Restore a disk image to a target volume:

`sudo asr restore --source {file image file with mandatory .dmg extension} --target {file volume file}`

- Erase the target volume before restoring:

`sudo asr restore --source {file image file with mandatory .dmg extension} --target {file volume file} --erase`

- Skip verification after restoring:

`sudo asr restore --source {file image file with mandatory .dmg extension} --target {file volume file} --noverify`

- Clone volumes without the use of an intermediate disk image:

`sudo asr restore --source {file volume file} --target {file volume file}`

```

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Same doubt here with volume_file.

File: pages/osx/asr.md [hunk 1]
```diff
@@ -6,16 +6,16 @@
 
 - Restore a disk image to a target volume:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}}`
+`sudo asr restore --source {{image_filename.dmg}} --target {{path/to/volume_file}}`
```
Comment (blueskyson) [inline pages/osx/asr.md:9]:
```suggestion
`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}}`
```

File: pages/osx/asr.md [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
 
 - Erase the target volume before restoring:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}} --erase`
+`sudo asr restore --source {{image_filename.dmg}} --target {{path/to/volume_file}} --erase`
```
Comment (blueskyson) [inline pages/osx/asr.md:13]:
```suggestion
`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}} --erase`
```

File: pages/osx/asr.md [hunk 3]
```diff
@@ -6,16 +6,16 @@
...
 
 - Skip verification after restoring:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}} --noverify`
+`sudo asr restore --source {{image_filename.dmg}} --target {{path/to/volume_file}} --noverify`
```
Comment (blueskyson) [inline pages/osx/asr.md:17]:
```suggestion
`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}} --noverify`
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**